app/admin.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class admin extends Model
```

Comment:
class names are supposed to be CamelCase and also file name

Reply:
admin.php was deleted